lua/null-ls/builtins/diagnostics/pylama.lua

```diff
@@ -15,26 +15,32 @@ return h.make_builtin({
         check_exit_code = function(code)
             return code < 1
         end,
-        on_output = h.diagnostics.from_json({
-            attributes = {
-                message = "message",
-                row = "lnum",
-                col = "col",
-                severity = "etype",
-                code = "number",
-                source = "source",
-            },
-            severities = {
-                E = h.diagnostics.severities["error"],
-                W = h.diagnostics.severities["warning"],
-                S = h.diagnostics.severities["warning"],
-                I = h.diagnostics.severities["warning"],
-                C = h.diagnostics.severities["warning"],
-                T = h.diagnostics.severities["warning"],
-                F = h.diagnostics.severities["information"],
-                D = h.diagnostics.severities["information"],
-                R = h.diagnostics.severities["information"],
-            },
-        }),
+        on_output = function(params)
+            if params.err == "failed to decode json: Expected value but found invalid token at character 1" then
```

Comment:
```suggestion
            if params.err:find("failed to decode json") then
```
I think it's a little better to match against a general JSON error to avoid this breaking again if the error format changes.

Reply:
Thanks. I wasn't sure how to do this so went with the more specific case. This is much better.